File: src/1935_MaximumNumberOfWordsYouCanType/1935.rb
# @param {String} text
# @param {String} broken_letters
# @return {Integer}
def can_be_typed_words(text, broken_letters)
    broken = broken_letters.split("")

    return text.split(" ").filter{|w| (w.split("") & broken).empty? }.count
end

File: src/1935_MaximumNumberOfWordsYouCanType/1935.go
func canBeTypedWords(text string, brokenLetters string) int {
    m := make(map[rune]bool) 
    
    for _, c := range brokenLetters {
        m[c] = true 
    }
    
    var not_count int = 0
    var size int = 0
    for _, word := range strings.Split(text, " ") {
        fmt.Println(word)
        size++
        for _, c := range word {
            if m[c] {
                not_count++
                break
            }
        }
    }

    return size - not_count
}